_posts/2022-07-07-release-ml-2.1.0.md

```diff
@@ -0,0 +1,159 @@
+algorithms to transmit and persist model data as unbounded data streams. 
+This release continues the effort by improving and validating the infrastructure
+needed to develop online learning algorithms.
+
+Specifically, this release added two online learning algorithm prototypes (i.e. 
+OnlineKMeans and OnlineLogisticRegression) with tests covering the entire 
+lifecycle of using these algorithms. These two algorithms introduce concepts 
+such as global batch size and model version, together with metrics and APIs to 
+set and get those values. While the online algorithm prototypes have not been 
+optimized for prediction accuracy yet, this line of work is an important step 
+toward setting up best practices for building online learning algorithms in 
+Flink ML. We hope more contributors from the community can join this effort.
+
+### Tools for benchmarking algorithm performance
+
+An easy-to-use benchmark framework is critical to developing and maintaining 
+performant algorithm libraries in Flink ML. This release added a benchmark 
+framework that provides APIs to write pluggable and reusable data generators, 
+takes benchmark configuration in JSON format, and outputs benchmark results in 
+JSON format to enable custom analysis. An off-the-shelf script is provided to 
+visualize benchmark results using Matplotlib. Feel free to check out this 
+[README](https://github.com/apache/flink-ml/blob/master/flink-ml-benchmark/README.md)
+for instructions on how to use this benchmark framework.
+
+The benchmark framework currently supports evaluating algorithm throughput. 
+In the future release, we plan to support evaluating algorithm latency and 
+accuracy.
+
+## Python SDK
+
+This release enhances the Python SDK so that operators in the Flink ML Python 
+library can invoke the corresponding operators in the Java library. The Python 
+operator is a thin-wrapper around the Java operator and delivers the same 
+performance as the Java operator during execution. This capability significantly
+improves developer velocity by allowing algorithm developers to maintain both 
+the Python and the Java libraries of algorithms without having to implement 
+those algorithms twice. 
+
+## Algorithm Library
+This release continues to extend the algorithm library in Flink ML, with the 
+focus on validating the functionalities and the performance of Flink ML 
+infrastructure using representative algorithms in different categories.
+
+Below are the lists of algorithms newly supported in this release, grouped by 
+their categories:
+
+- feature engineering (MinMaxScaler, StringIndexer, VectorAssembler, 
+StandardScaler, Bucketizer)
+- online learning (OnlineKmeans, OnlineLogisiticRegression)
+- regression (LinearRegression)
+- classification (LinearSVC)
+- evaluation (BinaryClassificationEvaluator)
+
+Example Python and Java programs for these algorithms are provided on the Apache
+Flink ML [website](https://nightlies.apache.org/flink/flink-ml-docs-stable/) to 
```

Comment:
Change the link to https://nightlies.apache.org/flink/flink-ml-docs-release-2.1/?